cranelift/codegen/src/isa/s390x/inst/emit.rs

```diff
@@ -2082,6 +2082,12 @@ impl MachInstEmit for Inst {
                 let opcode = 0xb904; // LGR
                 put(sink, &enc_rre(opcode, rd.to_reg(), rm));
             }
+            &Inst::MovPReg { rd, rm } => {
+                let rm: Reg = rm.into();
+                debug_assert!([regs::gpr(14), regs::gpr(15)].contains(&rm));
```

Comment:
I'd remove gpr(14), since this *is* actually allocatable.  And again preferably use stack_reg instead of hardcoding 15.